Base/res/ladybird/default-config/BrowserContentFilters.txt

```diff
@@ -791,19 +1091,33 @@ trx-hub.com
 turn.com
 tvpixel.com
 twiago.com
+twx.2mdn.net
+twx.doubleclick.net
+tyjyrkcrv.com
 tynt.com
+udc.yahoo.com
+udcm.yahoo.com
 ugdturner.com
+uk.2mdn.net
+ukrpts.net
+umekian.pw
 unblockia.com
 undertone.com
 uniqodo.com
 unrulymedia.com
 unrulyvideo.com
+upload.luckyorange.net
 upsellit.com
 urbanairship.com
 usabilla.com
 userreport.com
 userzoom.com
+uunyadgda1.doubleclick.net
+uunyadgds1.doubleclick.net
```

Comment:
Aren't these already filtered by the `doubleclick.net` entry? How did you consider new entries to this list?